modules/ROOT/pages/stats-monitoring-prometheus.adoc

```diff
@@ -22,14 +22,14 @@ include::partial$_define_stats_item_names.adoc[]
 
 Like other Couchbase services, Sync Gateway offers xref:stats-prometheus.adoc[Prometheus-compatible metrics] in a flat JSON format.
 
-Use the `_metrics` method to request these metrics:
+Use the `metrics` endpoint to request these metrics:
 
 [#ex-metprom]
 .Metrics in Prometheus format
 ====
 [source,console]
 ----
-curl -X GET "http://localhost:4986/_metrics" -H "accept: application/json"
+curl -X GET "http://localhost:4986/metrics" -H "accept: application/json"
```

Comment:
nit: always wondered about this for docs, should we omit `-X GET` from these? curl will do GET by default.

Reply:
Good point, but it seems to be the style throughout the Sync Gateway docs (AFAICT) and I don't want to introduce inconsistency

Reply:
We're talking about a REST API call, being explicit with the HTTP method is acceptable, and provides value to people who don't have a good mental model of `curl`'s defaults